templates/taxi/car_detail.html

```diff
@@ -19,4 +19,16 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+
+  {% if user in car.drivers.all %}
```

Comment:
The condition `if user in car.drivers.all` should be `if user in car.drivers.all()` to correctly call the `all()` method and check if the user is in the list of drivers.